### DISCUSSION TITLE
LF1 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#140] [Show and tell]
https://teambasedinquirylearning.github.io/precalculus/2024/LF1.html

Author: @tdegeorge 
Reviewer: @kathypinzon

**Comment by kathypinzon** [comment]
Activity 3.1.2 - "thermostat" is spelled wrong
The wording for part (c) is weird - maybe "how much the temperature was decreasing each one minute"

**Comment by kathypinzon** [comment]
Definition 3.1.6 I would split into two parts - a remark and the definition:

Remark: An average rate of change helps us to see and understand how a function is generally behaving. For example, in [Activity 3.1.2] and [Activity 3.1.5], we began to see how the temperature of Robert’s house was decreasing every minute the air conditioner was on. In other words, when looking at average rate of change, we are comparing how one quantity is changing with respect to something else changing.

Definition: An **average rate of change** "of a" function calculates the amount of change in one item divided by the corresponding amount of change in another.....

**Comment by kathypinzon** [comment]
3.1.7 a remove the "for" - I would also suggest adding in some MC options where slope is positive (like 7) maybe instead of -1/3
3.1.7 b add in a MC where slope is positive.
3.1.7 I would add another part that says something like "does this function have a constant average rate of change?"

**Comment by kathypinzon** [comment]
3.1.11 something weird happens in the graph at x=0... there is a hole or something.
3.1.11 part (b) you can't see the point at x=5 I know they have the equation so could use that but maybe mention it?

**Comment by kathypinzon** [comment]
3.1.13 instead of is use "has a constant"

**Comment by kathypinzon** [comment]
3.1.14 same weird thing is happening with the red graph at x=0
I would include MC options for parts (a)-(f)

**Comment by kathypinzon** [comment]
3.1.15 say "Slope is a way to describe the steepness of a line. The red..."

**Comment by kathypinzon** [comment]
3.1.18 I think MC options would be helpful.. it can be the same for both parts (a) and (b)

**Comment by kathypinzon** [comment]
3.1.19 put "slope" in a text box so not math mode.  Also, I would suggest using the letter m as well.
